site/content/en/docs/Installation/Creating Cluster/gke.md

```diff
@@ -167,6 +218,33 @@ gcloud container node-pools create windows \
 
 where [CLUSTER_NAME] is the name of the cluster you created.
 
+### Create an Autopilot mode cluster for Agones
+
+Agones supports Autopilot clusters in [alpha]({{<ref "/docs/Guides/feature-stages#alpha">}}).
+
+```bash
+gcloud container clusters create-auto [CLUSTER_NAME] \
+  --region=[COMPUTE_REGION] \
+  --release-channel=[RELEASE_CHANNEL] \
+  --autoprovisioning-network-tags=game-server \
+```
+Replace the following:
+
+* **CLUSTER_NAME**: the name of your cluster.
+* **COMPUTE_REGION**: the
+  [Google Cloud region](https://cloud.google.com/compute/docs/regions-zones#available)
+  for the cluster.
+* **RELEASE_CHANNEL**: the
+  [GKE release channel](https://cloud.google.com/kubernetes-engine/docs/release-notes#current_versions)
+  for the cluster. Can be one of `rapid`, `regular`, or `stable`. The default
+  is `regular`. The release channel must use version {{% k8s-version %}} or
```

Comment:
Instead of `The release channel must use version {{% k8s-version %}} or later` what about something like "Pick a release channel that uses a supported version of Kubernetes". We could also link to https://agones.dev/site/docs/installation/#usage-requirements. 

Also, I think with the change of {{% k8s-version %}} to now be a comma separated list of versions this won't read quite as well.